### PR TITLE
Loganj/breaker breaker

### PR DESCRIPTION
**Pull request by loganj**
Lots of breaking name changes.

### CODE REVIEW

File: flow-sample/src/main/java/com/example/flow/view/FriendListView.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public void setFriends(List<User> friends) {
   }
 
   @Override public void updateSelection() {
-    Paths.FriendPath screen = (Paths.FriendPath) Flow.get(getContext()).getBackstack().top();
+    Paths.FriendPath screen = (Paths.FriendPath) Flow.get(getContext()).getHistory().top();
```
Comment (rjrjr) [inline flow-sample/src/main/java/com/example/flow/view/FriendListView.java]: Weren't you going to make this <T> T top()?

File: flow/src/main/java/flow/Flow.java [hunk 2]
```diff
@@ -235,6 +235,33 @@ private void move(PendingTraversal pendingTraversal) {
     }
   }
 
+  private static History preserveEquivalentPrefix(History current, History proposed) {
```
Comment (rjrjr) [inline flow/src/main/java/flow/Flow.java]: Sure you don't want this to be public?

Comment (rjrjr) [inline flow/src/main/java/flow/Flow.java]: I guess not.

File: flow-sample/src/main/java/com/example/flow/view/FriendListView.java (hunk 1 above)
Comment (loganj) [inline flow-sample/src/main/java/com/example/flow/view/FriendListView.java]: I did, this is noise.

**Pull request merged**